### ISSUE TITLE
the inference of baseline

### ISSUE THREAD

**Issue by wang-zm18** [MiniEval/citl#2]
Hello, Clinton, nice work! Could you please proide the source of baselines shown in the paper for the convenience of  result reproduction, such as BERT and MAE. Thank you in advance!

**Comment by wang-zm18** [comment]
Another question: I have tried the reproduction on the CMU swim subset using your provided pre-trained checkpoints. However, I got the results under interval=5：
                                                                     L2P: 0.0968, L2Q: 0.2944, NPSS: 0.2934
The corresponding results shown in the paper are: 
                                                                     L2P: 0.134, L2Q: 0.212, NPSS: 0.186
Is there some data process that I missed?  Thank you in advance!

**Comment by MiniEval** [comment]
The official implementation is reworked from the original experimental setup, largely for readability and usability reasons. As such, our experimental implementation for BERT and MAE baselines are not compatible with the official release.

As for the difference in results, a couple changes did occur during the transition:

1. The world scale of the CMU dataset was changed from scaling by standard deviation (i.e. approximately x0.0875) to metre units (x 0.01 * 2.54 / 0.45).
2. The dataloader is set up differently, which causes the test samples used to be different as well. This can have a notable effect on the swim dataset's (i.e. categories 125&126) metrics.

The implementation is still correct. I've checked the performance of the official implementation with LERP and the model does have the expected performance improvement.

LERP (5f) - L2P: 0.0836, L2Q: 0.1929, NPSS: 0.3410
CITL (5f) - L2P: 0.1226, L2Q: 0.2436, NPSS: 0.3003
LERP (15f) - L2P: 0.3321, L2Q: 0.6741, NPSS: 1.4175
CITL (15f) - L2P: 0.2456, L2Q: 0.5199, NPSS: 0.7727
